styles.css

```diff
@@ -305,4 +314,112 @@ main {
   position: absolute;
   margin-left: 180px;
   margin-top: 187px;
-}
\ No newline at end of file
+}
+
+.formsection {
+  display: grid;
+  grid-template-columns: 320px;
+  grid-template-rows: 830px;
+  margin-top: 16px;
+}
+
+.getstarted {
+  position: absolute;
+  width: 249px;
+  height: 50px;
+  left: 16px;
+  margin-top: 80px;
+  font-family: 'Playfair Display', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 45px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.main {
+  position: absolute;
+  width: 288px;
+  height: 63px;
+  left: 16px;
+  margin-top: 130px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+footer {
+  position: absolute;
+  width: 320px;
+  height: 150px;
+  margin-top: 4630px;
+  margin-left: 0;
+  background: #1a2236;
+}
+
+.pa {
+  position: absolute;
+  width: 127px;
+  height: 24px;
+  left: 16px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #92dcdc;
+  margin-top: 30px;
+}
+
+.arrowbox {
+  position: absolute;
+  width: 22px;
+  height: 22px;
+  left: 140px;
+  margin-top: 32px;
+}
+
+.socialmediaicons {
+  margin-top: 90px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.form {
+  margin-top: 230px;
+  margin-left: 16px;
+}
+
+.form form {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 20px;
+}
+
+form input {
+  width: 288px;
+  height: 52px;
+}
+
+form textarea {
+  width: 288px;
+  height: 112px;
+}
+
+form #submit {
+  color: #fff;
+  background: #1a2236;
+  cursor: pointer;
+}
+
+form #submit:hover {
+  border: 2px solid #00f;
+}
```

Comment:
- [ ] While you did a wonderful job styling your project :+1: you need to have a responsive design until the desktop version, which means that your design shouldn't have a fixed width. As the previous reviewer asked kindly check out the [responsive design](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#responsive-design) section in this link for more info. Doing this will also help you when you get to the desktop version milestone of your project. so it is necessary that you may consider fixing that now :+1: